src/cpp/session/modules/SessionFind.cpp

```diff
@@ -1181,16 +1267,17 @@ core::Error completeReplace(const json::JsonRpcRequest& request,
                                   &ignoreCase,
                                   &directory,
                                   &filePatterns,
+                                  &excludeFilePatterns,
                                   &originalFindCount,
-                                  &replacePattern,
-                                  &useGitIgnore);
+                                  &replacePattern);
    if (error)
       return error;
 
    static const int kUpdatePercent = 5;
    LocalProgress* pProgress = new LocalProgress(originalFindCount, kUpdatePercent);
```

Comment:
Sure. I think either way a comment would clear it up, since 5 was chosen somewhat arbitrarily (presumably because it seemed to have the best balance of feeling responsive but not being overly frequent?). If you did any testing that led to the choice of 5%, I think it would be good to note it in the code! :)

Reply:
Ok added a comment